pkgs/development/libraries/cdo/default.nix

```diff
@@ -0,0 +1,60 @@
+{ stdenv, fetchurl, curl
+, enable_data ? true        # DATA support [default=yes]
```

Comment:
Note that all these parameters are not discoverable. Packages in Nixpkgs are usually configured with lowest-common denominator in mind, using params only sparingly. Reifying every `configure` switch as a parameter is certainly going overboard in any case.

Please consider reducing this to only cover options users are likely to want to change or for features that are particularly costly in terms of build footprint.